pkgs/top-level/python-packages.nix

```diff
@@ -24461,6 +24461,10 @@ EOF
   parse-type = callPackage ../development/python-modules/parse-type { };
 
   ephem = callPackage ../development/python-modules/ephem { };
+
+  argon2_cffi = callPackage ../development/python-modules/argon2_cffi { };
+
+  pytest-timeout = callPackage ../development/python-modules/pytest-timeout { };
```

Comment:
@FRidh does not like this :)

Reply:
`python-packages` is fubar, with multiple beginnings and endings

Reply:
I don't like it? Well, we should aim at having ordering. PR's that improve ordering are more than welcome.

Reply:
I will have a look into it, but not for this PR ;)

Reply:
Even partial order should help to avoid merge conflicts.

Reply:
I just tried my best to add them somewhere ...